packages/lib/getBusyTimes.ts

```diff
@@ -179,13 +179,24 @@ const _getBusyTimes = async (params: {
   performance.measure(`prisma booking get took $1'`, "prismaBookingGetStart", "prismaBookingGetEnd");
   if (credentials?.length > 0 && !bypassBusyCalendarTimes) {
     const startConnectedCalendarsGet = performance.now();
-    const calendarBusyTimes = await getBusyCalendarTimes(
+
+    const calendarBusyTimesQuery = await getBusyCalendarTimes(
       credentials,
       startTime,
       endTime,
       selectedCalendars,
       shouldServeCache
     );
+
+    if (!calendarBusyTimesQuery.success) {
```

Comment:
We throw an error when failing to get the calendar event times because it's not a true representation of a user's busy times.